### PR TITLE
[DUOS-1175][risk=no] Minor cleanup/refactoring

### PR DESCRIPTION
**Pull request by rushtong**
## Addresses
https://broadworkbench.atlassian.net/browse/DUOS-1175

## Changes
* Move constants to a function
* Move all functions to anonymous functions
* Clean up formatting a bit

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] PR is labeled with a Jira ticket number and includes a link to the ticket
- [ ] PR is labeled with a security risk modifier [no, low, medium, high] 
- [ ] PR describes scope of changes

In all cases:

- [ ] Get a minimum of one thumbs worth of review, preferably 2 if enough team members are available
- [ ] Get PO sign-off for all non-trivial UI or workflow changes
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/ResearcherProfile.js [hunk 1]
```diff
@@ -375,28 +356,28 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
     return obj;
   };
 
-  saveProperties(profile) {
+  saveProperties = async (profile) => {
     Researcher.createProperties(profile).then(()=> {
```
Comment (JVThomas) [inline src/pages/ResearcherProfile.js]: If you're modifying this to be an async/await function then you can take this a step further and get rid of the ```then``` blocks.

```
await Researcher.createProperties(profile)
await this.saveUser()
this.setState({...
```

File: src/pages/ResearcherProfile.js [hunk 2]
```diff
@@ -465,7 +446,28 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
     return _.omit(_.cloneDeep(profile), ['libraryCards', 'libraryCardEntries']);
   };
 
+  generateCountryNames = () => {
```
Comment (JVThomas) [inline src/pages/ResearcherProfile.js:435]: Good call with the helper functions, keeps the variables contained and sharable when needed.

File: src/pages/ResearcherProfile.js [hunk 3]
```diff
@@ -406,17 +387,17 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
     }
   };
 
-  updateResearcher(profile) {
+  updateResearcher = (profile) => {
```
Comment (JVThomas) [inline src/pages/ResearcherProfile.js]: This can be updated with await/async notation like createResearcher

**Review by JVThomas — CHANGES_REQUESTED** [review]

**Review by raejohanek — APPROVED** [review]
the refactoring on the page looks great and it all works well

File: src/pages/ResearcherProfile.js [hunk 4]
```diff
@@ -406,17 +377,17 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
     }
   };
 
-  updateResearcher(profile) {
+  updateResearcher = async (profile) => {
     const profileClone = this.cloneProfile(profile);
     Researcher.updateProperties(this.state.currentUser.dacUserId, true, profileClone).then(()=> {
```
Comment (JVThomas) [inline src/pages/ResearcherProfile.js]: The nested ```.then()``` chains can be cleaned up with await statements like in ```saveProperties```

File: src/pages/ResearcherProfile.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-import _, { isNil, isEmpty } from 'lodash';
+import {omit, cloneDeep, isEmpty, isNil, get, trim, omitBy } from 'lodash';
```
Comment (JVThomas) [inline src/pages/ResearcherProfile.js:1]: There's a second lodash import statement on line 13 where it's being aliased as ```ld```, so the entire library is still being imported.

The ```ld``` alias is being used on lines 463 and 572

**Review by JVThomas — CHANGES_REQUESTED** [review]

File: src/pages/ResearcherProfile.js (hunk 4 above)
Comment (rushtong) [reply src/pages/ResearcherProfile.js]: Good point, will address

File: src/pages/ResearcherProfile.js (hunk 5 above)
Comment (rushtong) [reply src/pages/ResearcherProfile.js:1]: Totally missed that ... will clean that up too.

**Review by JVThomas — APPROVED** [review]
PR is good to go!

**Pull request merged**